jobs/e2e_node/image-config-beta.yaml

```diff
@@ -0,0 +1,19 @@
+# To copy an image between projects:
```

Comment:
This file is just a copy of `image-config.yaml`.

Reply:
ping @yguo0905